src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, Dispatch, SetStateAction } from 'react';
+import { Divider, List, ListItem, ListItemText, ListItemIcon, Box, CircularProgress, ListSubheader } from '@mui/material';
+import { Link } from 'react-router-dom';
+
+import useStyles from './styles';
+
+import githubWhite from '../assets/logo/github_white.png';
+// import githubblack from '../assets/logo/github_black.png';
```

Comment:
Commented out code

Reply:
I am intending to use the GitHub black logo when I will be Implementing dark mode.